Objects/longobject.c

```diff
@@ -6291,7 +6291,7 @@ int.to_bytes
         the most significant byte is at the beginning of the byte array.  If
         byteorder is 'little', the most significant byte is at the end of the
         byte array.  To request the native byte order of the host system, use
-        `sys.byteorder' as the byte order value.  Default is to use 'big'.
+        `sys.byteorder` as the byte order value.  Default is to use 'big'.
```

Comment:
Wouldn't single quotes also cause Sphinx warnings from using the default role? I might prefer to just remove the quotes:
```suggestion
        sys.byteorder as the byte order value.  Default is to use 'big'.
```
Other docstrings generally don't use code markup either. For example:

```
>>> print(os.lstat.__doc__)
Perform a stat system call on the given path, without following symbolic links.

Like stat(), but do not follow symbolic links.
Equivalent to stat(path, follow_symlinks=False).
```

Reply:
Happy to remove these. Note that they are backticks and not single quotes. However, I think you are right and we should not use single backticks in the default role according to https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html double backticks are probably fine but removing is probably simpler.

 Single quotes are used for big and little. Should we also remove those?

Reply:
I suppose I was using "quotes" as a general term here :)

I think the single quotes for `'big'` and little make sense to keep because those refer to strings. I would keep them unless they cause problems somehow.

Reply:
@JelleZijlstra Thanks. I have removed the backticks completely as suggested.